README.md

```diff
@@ -100,13 +163,15 @@ Now Prettier will be run every time the file is saved.
 
 ### VSCode
 
-VSCode is not familiar with the solidity language, so [`solidity support`](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity) needs to be installed.
+VSCode is not familiar with the solidity language, so [`solidity`](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity) support needs to be installed.
 
 ```Bash
 code --install-extension JuanBlanco.solidity
 ```
 
-Having done that you should proceed to install [`prettier-vscode`](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+This extension provides a basic integration with Prettier for most cases no further action is needed.
+
+If you want more control on other details, you should proceed to install [`prettier-vscode`](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
```

Comment:
I'd use `over` instead of `one` as in "more control over"

Reply:
done